src/Microsoft.AspNetCore.Blazor.Build/targets/RazorCompilation.targets

```diff
@@ -39,18 +39,18 @@
     <ProjectCapability Include="Blazor" />
     <ProjectCapability Include="DotNetCoreRazorConfiguration" />
     
-    <RazorConfiguration Include="Blazor-0.1">
-      <Extensions>Blazor-0.1;Blazor.AngleSharp-0.1;$(CustomRazorExtension)</Extensions>
+    <RazorConfiguration Include="Blazor-0.2">
+      <Extensions>Blazor-0.2;Blazor.AngleSharp-0.2;$(CustomRazorExtension)</Extensions>
```

Comment:
To ease future updates, should we be moving all these version numbers out into a property in some shared location?